src/components/CheckBox/CheckBox.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import CheckedList from './CheckedList';
+
+const CheckBox = () => {
+  const [isClick, setIsClick] = useState(0);
+  const handleClick = id => {
+    setIsClick(id - 1);
+  };
+
+  // const location = useLocation();
+
+  // parseQs
+  // const arrived = parseLocation(location);
+
+  return (
+    <div>
+      <CheckBoxContainer>
+        <SortAirplane>항공사</SortAirplane>
+        <AlignBtn>
+          {SELECT_ALL.map(({ id, name }, index) => (
+            <Btn
+              onClick={() => handleClick(id)}
+              key={id}
+              active={isClick === index}
+            >
+              {name}
+            </Btn>
+          ))}
+        </AlignBtn>
+        <SelectUl>
+          <CheckedList />
+        </SelectUl>
+      </CheckBoxContainer>
+    </div>
+  );
+};
+
+const CheckBoxContainer = styled.div`
+  width: 200px;
+  height: auto;
+  border-top: ${props => props.theme.borders.borderBox};
+  border-bottom: ${props => props.theme.borders.borderBox};
+  margin: 8px 16px 0 0;
+`;
+
+const SortAirplane = styled.div`
+  margin: 8px 0;
+`;
+
+const AlignBtn = styled.div`
+  ${props => props.theme.flex.flexBox('row', 'center', 'flex-start')}
```

Comment:
잘 적용하셨네요!  row는 디폴트 값이기 때문에 ''으로 표현할 수 도 있습니다!
이 부분은 경연님이 더 편하다라고 생각하시는 방법으로 사용하시면 됩니다!
```suggestion
  ${({theme})=> theme.flex.flexBox('', 'center', 'flex-start')}
```